src/lambdaisland/embedkit.clj

```diff
@@ -159,7 +159,7 @@
         ~@body
         (catch clojure.lang.ExceptionInfo e#
           (if (and (= 401 (:status (ex-data e#)))
-                   (atom? conn#)
+                   (instance? clojure.lang.Atom conn#)
```

Comment:
There is no `atom?` in standard library. And the `atom?` in `clojure.inspect` is not what I want.